### ISSUE TITLE
White screen flashes on article load or change in dark mode.

### ISSUE THREAD

**Issue by Death-Pact** [yang991178/fluent-reader-lite#79]
**Describe the bug**
Whenever selecting feed articles, or cycling to next or previous article, app flashes a bright white screen.

**To Reproduce**
1. load apk
2. Click on any feed or article
3. white screen blasts eyeballs
4. article finishes loading in dark mode.

**Expected behavior**
screen should stay black during article load, selection, or change.

**Platform (please complete the following information):**
 - OS: Android 12
 - App Version 1.0.3

**Comment by butjer** [comment]
Same for me. Would be very nice if this flashing could be removed.

**Comment by spackmat** [comment]
Yeah, that "white screen blasts eyeballs" thing is really annoying.
